core/src/main/java/tc/oc/pgm/scoreboard/SidebarMatchModule.java

```diff
@@ -312,6 +312,14 @@ private Component renderTitle(final Config config, final MapInfo map) {
       }
     }
 
+    // Display "Blitz: Rage" rather than "Blitz and Rage"
+    if (map.getTags().contains(new MapTag("blitz", "Blitz", true, true))
+        && map.getTags().contains(new MapTag("rage", "Rage", true, true))) {
+      games.add(translatable("gamemode.br.name").color(NamedTextColor.AQUA));
+      games.remove(translatable("gamemode.blitz.name").color(NamedTextColor.AQUA));
+      games.remove(translatable("gamemode.rage.name").color(NamedTextColor.AQUA));
+    }
```

Comment:
In total there will never be more than 2 games in the list, due to how `games.size() < 2` is the only thing that adds auxiliary tags (otherwise breaks) and primary gamemodes breaks with "generic" if multiple primaries are present. This means you can simply clear the games list and add blitz rage directly. 
Additionally, i think creating dummy objects just so they match what you want to remove, or what you want to "contains"-check is generally bad for what are essentially singleton objects (the map tags) which could very well be matched by identity-check, i think it's better to match by checking the id directly.

```suggestion
    if (games.size() == 2 && Stream.of("blitz", "rage").allMatch(
            id-> map.getTags().stream().anyMatch(mt -> mt.getId().equals(id))) {
      games.clear();
      games.add(translatable("gamemode.br.name").color(NamedTextColor.AQUA));
    }
```